Code/GraphMol/MolDraw2D/test1.cpp

```diff
@@ -4170,6 +4172,36 @@ void testGithub4156() {
   std::cerr << " Done" << std::endl;
 }
 
+void test23JSONAtomColourPalette() {
+  std::cerr << " ----------------- Test JSON atomColourPalette" << std::endl;
+  {
+    auto m = "c1cncs1"_smiles;
+    TEST_ASSERT(m);
+    MolDraw2DUtils::prepareMolForDrawing(*m);
+    MolDraw2DSVG drawer(250, 200);
+    const char *json =
+        "{\"atomColourPalette\": {\"7\": [0.2, 0.4, 0.9], \"16\": [0.9, 0.6, 0.0]}, "
+        "\"rotate\": 90, "
+        "\"bondLineWidth\": 5}";
+    MolDraw2DUtils::updateDrawerParamsFromJSON(drawer, json);
```

Comment:
I know the ugly old-school string-literal syntax is used elsewhere in this file, but for new code I think it's nice to use the new syntax
```suggestion
    const char *json =
        R"JSON({"atomColourPalette": {"7": [0.2, 0.4, 0.9], "16": [0.9, 0.6, 0.0]},
        "rotate": 90, 
        "bondLineWidth": 5})JSON";
    MolDraw2DUtils::updateDrawerParamsFromJSON(drawer, json);
```